WalletWasabi.Gui/Global.cs

```diff
@@ -510,6 +510,15 @@ private async Task AddKnownBitcoinFullNodeAsHiddenServiceAsync(AddressManager ad
 
 		public async Task InitializeWalletServiceAsync(KeyManager keyManager)
 		{
+			var walletService = await CreateWalletServiceAsync(keyManager);
+
+			WalletService = walletService;
```

Comment:
Why not inline this?
`WalletService = await CreateWalletServiceAsync(keyManager);`